float32 left_current    # [A] Current drawn by left motor
float32 left_voltage    # [V] Terminal voltage on left motor
float32 right_current   # [A] Current drawn by right motor
float32 right_voltage   # [V] Terminal voltage on right motor
